### ISSUE TITLE
synthetic data

### ISSUE THREAD

**Issue by centurions** [sirius-ai/LPRNet_Pytorch#14]
Hi thank you for your great work .
Can I use synthetic data for training and predict on real images ?

**Comment by sirius-ai** [comment]
Yes, it depend on the data distribution how like in synthetic data and real images .
